test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1

```diff
@@ -13,7 +13,7 @@ Describe 'Clipboard cmdlet tests' -Tag CI {
         }
```

Comment:
@iSazonov, your last commit had 52 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path.Get-ChildItem -Path
```powershell
Expected 4, but got 5.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 46
46:             $result.Count | Should -Be 4
```
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path.Get-ChildItem -Path -Hidden
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 52
52:             $result.Count | Should -Be 1
```
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path.Get-ChildItem -Path -Attribute Hidden
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 59
59:             $result.Count | Should -Be 1
```
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path -Directory/-File.Get-ChildItem -Path -File
```powershell
Expected 2, but got 3.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 79
79:             $result.Count | Should -Be 2
```
Extended FileSystem Provider Tests for Get-ChildItem cmdlet.Validate Get-ChildItem -Path -Directory/-File.Get-ChildItem -Path -File -Hidden
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystemProviderExtended.Tests.ps1: line 84
84:             $result.Count | Should -Be 1
```